### ISSUE TITLE
Support for dotnet-runtime > 3.0

### ISSUE THREAD

**Issue by ar2-de2** [arkane-systems/genie#60]
```
 # apt install systemd-genie
Reading package lists... Done
Building dependency tree
Reading state information... Done
...
The following packages have unmet dependencies:
systemd-genie : Depends: dotnet-runtime-3.0 but it is not installable
E: Unable to correct problems, you have held broken packages.
 # dpkg -l | grep dotnet-runtime
ii  dotnet-runtime-3.1  3.1.5-1  amd64  Microsoft .NET Core Runtime - 3.1.5 Microsoft.NETCore.App 3.1.5
ii  dotnet-runtime-deps-3.1  3.1.5-1  amd64  dotnet-runtime-deps-3.1 3.1.5
```

**Comment by ar2-de2** [comment]
So, the problem seems like "chicken or the egg" — **dotnet-runtime-3.0** now can be installed with **snap**, **snap** can only be runned with **systemd**.

**Comment by cerebrate** [comment]
This will be fixed in the next release (1.24) to support Ubuntu Focal.
